### PR TITLE
DOC: tweaks to firing rate example

### PR DESCRIPTION
**Pull request by jasmainak**
@ntolley just some tiny tweaks. Is this good by you?

### CODE REVIEW

**Comment by jasmainak** [comment]
https://610-168215891-gh.circle-artifacts.com/0/html/auto_examples/plot_firing_pattern.html#sphx-glr-auto-examples-plot-firing-pattern-py

File: examples/plot_firing_pattern.py [hunk 1]
```diff
@@ -54,24 +55,15 @@
 plt.show()
 
 ###############################################################################
-# Let's do this for the rest of the cell types
-fig, axes = plt.subplots(1, 2, sharey=True, figsize=(8, 4))
-for gid, ax in zip([gid_dict['L2_pyramidal'][0],
-                    gid_dict['L5_pyramidal'][0]], axes):
-    ax.plot(times, vsoma[gid])
-    ax.set_title('%s (gid=%d)' % (net.gid_to_type(gid), gid))
```
Comment (ntolley) [inline examples/plot_firing_pattern.py:62]: It'd be nice to keep an example of `gid_to_type()`, unless it's used in another example?

Comment (jasmainak) [reply examples/plot_firing_pattern.py:62]: It's actually above: https://github.com/jonescompneurolab/hnn-core/pull/206/files#diff-a39b3b79040c9c527504aea86494f6d09a2716ff1b130fdc3845d5485b7d48e6R52

Comment (ntolley) [reply examples/plot_firing_pattern.py:62]: Ah, hidden for people too lazy to check the original.

Yeah this looks much more concise and clean, definitely merge unless you've got other ideas to tack on.

Comment (jasmainak) [reply examples/plot_firing_pattern.py:62]: also the colors match between different plots and the time axis is common. That was the main purpose :)

go ahead and merge if you're happy!

**Pull request merged**